lmfit/confidence.py

```diff
@@ -352,8 +352,8 @@ def calc_prob(self, para, val, offset=0., restore=False):
         return prob - offset
 
 
-def conf_interval2d(minimizer, result, x_name, y_name, nx=10, ny=10,
-                    limits=None, prob_func=None):
+def conf_interval2d(minimizer, result, x_name, y_name, nx=10, ny=10, nsigma=5,
+                    limits=None, prob_func=None, chi2_out=False):
```

Comment:
the addition of `nsigma` before the arguments that were present already would break backwards-compatibility. It would thus be better to add new arguments at the end of the list - could we move `nsigma` to be after `prob_func`?

Reply:
yes, will do.